src/TestFramework/MSTest.Core/Assertions/Assert.Boolean.cs

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !NETCOREAPP3_0_OR_GREATER && !NET6_0_OR_GREATER
+#define HIDE_MESSAGELESS_IMPLEMENTATION
+#endif
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+/// <summary>
+/// A collection of helper classes to test various conditions within
+/// unit tests. If the condition being tested is not met, an exception
+/// is thrown.
+/// </summary>
+public sealed partial class Assert
```

Comment:
We should probably follow the pattern used for other files (i.e. rename this file as `Assert.IsTrue`).